docs/local-dev-setup.md

```diff
@@ -2,53 +2,32 @@
 
 ?> _TODO_: update the content to work with modules in `terraform-ibm-modules` in GitHub.
 
-Follow these steps to set up your local development environment for Project GoldenEye.
+Follow these steps to set up your local development environment for working with the `terraform-ibm-modules` project.
+
+For regular maintenance tasks or to run certain pre-commit hooks manually, see [Regular developer tasks](dev-maintenance.md).
 
 ## Before you begin :id=dev-setup-prereqs
 
 - On macOS and Linux systems, [install](https://docs.brew.sh/Installation) or [update](https://docs.brew.sh/FAQ) Homebrew.
 - On Microsoft Windows, set up Windows Subsystem for Linux [WSL](https://ubuntu.com/wsl) and run commands within WSL.
 
-For regular maintenance tasks or to run certain pre-commit hooks manually, see [Regular developer tasks](/docs/solution-as-code?topic=solution-as-code-dev-maintenance).
-
-## Set the protocol for remotes and submodules :id=dev-setup-protocol
-
-By default, GoldenEye uses HTTPS to connect to remote repositories on GitHub Enterprise (github.ibm.com) and to pull in Terraform submodules. To avoid the prompt for credentials when you connect to a remote (for example, working with Git submodules or importing Terraform modules), choose one of the following options:
-
-- Update your local Git config file to use SSH for access to github.ibm.com/GoldenEye.
-
-    ```bash
-    git config --global url."git@github.ibm.com:GoldenEye".insteadOf "https://github.ibm.com/GoldenEye"
-    ```
-
-    Or
-
-- Create a `.netrc` file that includes your Git token to continue to use HTTPS.
-
-    ```bash
-    export GH_TOKEN=XXX
-    echo -e "machine github.ibm.com\n  login $GH_TOKEN" >> ~/.netrc
-    ```
-
-    Where `XXX` is the value of your Git token.
-
 ## Clone and initialize the repo :id=dev-setup-clone
 
 1.  Clone the repo you want to start working with:
 
-    - To clone by using SSH, run the following command. You need a valid SSH key for github.ibm.com. For more information, see the steps in the [GitHub docs](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
+    - To clone by using SSH, run the following command. You need a valid SSH key for github.com. For more information, see the steps in the [GitHub docs](https://docs.github.com/en/authentication/connecting-to-github-with-ssh).
 
         ```bash
-        git clone git@github.ibm.com:GoldenEye/<REPO_NAME>.git
+        git clone git@github.com:terraform-ibm-modules/<REPO_NAME>.git
         ```
 
     - To clone by using HTTPS, run the following command.
 
         ```bash
-        git clone https://github.ibm.com:GoldenEye/<REPO_NAME>.git
+        git clone https://github.ibm.com:terraform-ibm-modules/<REPO_NAME>.git
```

Comment:
It should be a slash in the https url, instead of a : here.

git clone https://github.ibm.com/terraform-ibm-modules/<REPO_NAME>.git